### DISCUSSION TITLE
Plugins - Bundles and Composites

### DISCUSSION THREAD

**Discussion by luiscreates** [XiaoFaye/WooCommerce.NET#740] [Q&A]
Hello,

Per the woocommerce documentation, there are a few plugins that extend certain objects such as product and order. 

For example, [product bundles](https://woocommerce.com/document/bundles-rest-api-reference/) and [composites](https://woocommerce.com/document/composite-products-rest-api-reference/). 

Are there any plans to extend this project to include such features? If not, I'll code away and submit a pull request.

Thanks!

**Comment by XiaoFaye** [comment]
Hi, 

Standard version of WooCommerce.NET will not implement any plugins, feel free to fork you own version and implement it.

Thanks,

**Comment by luiscreates** [comment]
Leaving this for anyone who is looking to use product bundles with this library. I've forked the base library and added the product bundles feature to support the extension. I'll be doing the same for composites too.

Forked repo can be found here: https://github.com/luiscreates/WooCommerce.NET.Extensions

Thanks James for this great library!
